CONTRIBUTING.md

```diff
@@ -184,6 +185,12 @@ Once you know that your changes have been accepted to the canonical repository,
   ```console
   $ git push fork :<branchname>
   ```
+## Current repository standards and configuration
```

Comment:
Which standards and which configuration do you have in mind here?

Reply:
this change was suggested by ocramius because i asked for a place where you can find a current template for repo configurations. (https://laminas.slack.com/archives/C4R13GTRD/p1617030866068200)

It turned out that there is a corresponding repo that tries to keep it up to date. There you will find information on what coding standards, qa tool settings or the like have to be observed.

In order to document this for other contributors in the future, I have added this link to this file with a note.

Reply:
This all fine but it doesn't answer the question: which standards and which configuration are meant here.
In other words, the following text is too general:

> To get an overview of which current standards and configurations should be used for the components…

Reply:
@nusphere
Do you have time to update your pull request to clarify which standards and which configuration are meant here?
Thanks in advance! 👍

Reply:
tomorrow, i will give it a try to update it

Reply:
Think this should be closed,@nusphere.